src/exports.jl

```diff
@@ -1,7 +1,9 @@
 function walkmodules(f, x::Module)
     f(x)
-    for n in names(x; all=true)
-        if isdefined(x, n)
+    for n in names(x; all = true, imported = true)
```

Comment:
Since we check `parentmodule(y) === x` below, `imported = true` here does not seem to be required (and causes extra works).  Or is there a reason that I miss for adding this?

```suggestion
    for n in names(x; all=true)
```

Reply:
I'll go ahead and merge this for now. Thanks for the patch!

Reply:
Thanks !